python/ray/autoscaler/v2/tests/test_utils.py

```diff
@@ -563,8 +563,8 @@ def test_cluster_status_formatter():
 Resources
 --------------------------------------------------------
 Total Usage:
- 0.5/3.0 CPU
```

Comment:
the old one actually looks better to me. 

@scottsun94 could you maybe chime in here - what's the best way to show decimal places in your opinion?

Reply:
Hmm. It does feel that something is broken when I see something like `0.01/4 GPU`

I don't know what's the best solution here. What if we only keep the decimal places the same/consistent before and after `/` on the same row?

Something like?
```
1/3 CPU
1.01/4.00 GPU


0.5/3.0 CPU
0.01/2.00 GPU

2/4 CPU
0.51/1.00 GPU

0/1 CPU
0/2 GPU
```